From fe36bf6f0dc1f0c1e927850dd78093cfe3b7aae2 Mon Sep 17 00:00:00 2001
From: Albert Astals Cid <aacid@kde.org>
Date: Sun, 28 Jul 2019 00:07:33 +0200
Subject: Fix compile on CI

src/workpackage/view.h:148:31: error: field m_scheduleActions has incomplete type QMap<QAction*, KPlato::Schedule*>
---
 src/workpackage/view.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/workpackage/view.h b/src/workpackage/view.h
index 6cae32b..cd2fc4e 100644
--- a/src/workpackage/view.h
+++ b/src/workpackage/view.h
@@ -24,6 +24,7 @@
 
 #include <KoView.h>
 
+#include <QMap>
 #include <QStackedWidget>
 
 class QPrinter;
-- 
cgit v1.1
